src/App.tsx

```diff
@@ -15,36 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphaberticaly = 'Sort alphabetically',
+  Length = 'Sort by length',
+  Reverse = 'Reverse',
+}
```

Comment:
The `SortType` enum values are not used directly in the code, but rather through constants such as `SORT_FIELD_ALPHABET`. Consider using the enum values directly in the code to make it cleaner and more consistent with typical TypeScript practices.